news/2021-09-26-project-loved-september-2021.md

```diff
@@ -0,0 +1,385 @@
+
+[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/1262927.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Camellia vs. Tsukasa Kaminose - Gunkyouzouteki Soseiriron](https://osu.ppy.sh/beatmapsets/1262927#mania) by [\_Stan](https://osu.ppy.sh/users/1653229), [\[Emiria\]](https://osu.ppy.sh/users/2046044), [biemote](https://osu.ppy.sh/users/799606), [Kyousuke-](https://osu.ppy.sh/users/2616570) and [Penguinosity](https://osu.ppy.sh/users/10235296)\
+160 BPM, 4:40 | 5.48★, 5.55★, 5.57★, 5.88★, 6.20★\
+*description written by [Penguinosity](https://osu.ppy.sh/users/10235296)*
+
+\_Stan makes their first appearance to Loved, with their 5 diff collaborative set of Gunkyouzouteki Soseiriron! Stan is a hardworking charter, which I'm sure many people who keep up with ranked charts should be familiar with. From the eclectic unapologetic patterning of \[Dream to Nightmare](https://osu.ppy.sh/beatmapsets/527728#mania/1125576) to the creative and diverse hybrid charting of [Luin of Celtchar](https://osu.ppy.sh/beatmapsets/1281422#mania/2879332), Stan has proven to be an essential member of the community over the years and has even contributed to Ranked standard sets! To any unfamiliar, *Gunkyou* (which I will call the song for short) is a chaotic and intense piano-based track off of Camellia's *Xroniel Xero* album and features a collaboration between him and the equally legendary Tsukasa Kaminose. Onto the set, though, a collaboration between these two artists should be charted as a full collaboration between a myriad of different charters. Those charters include Stan, Kyousuke-, biemote, [Emiria\] (otherwise known as victorica_db), and myself. Besides all of us simply having GDs on the set, the catch is that each of the 5 difficulties is of a different key count. These diffs go from 4K up to 8K, and all of what's here will give players a different experience from one map to the next. From the full rice approach to the 8K to the hybrid centered patterning of the 7 and 4K, each approach to the song brings something distinct to the table. Seeing a well-constructed key spread is rare enough these days, let alone one put together in a way that's been done here. We're both thankful for Stan's contributions to the community ove
```

Comment:
> From the full rice approach to the 8K

i don't know what rice is here and thus have no way of checking, but i assume this is supposed to be "From the full rice approach *of* the 8K"

i've also completed the unfinished last sentence using ~~ai autocomplete~~ some slight guesswork

```suggestion
\_Stan makes their first appearance to Loved, with their 5-diff collaborative set of Gunkyouzouteki Soseiriron! Stan is a hardworking charter, which I'm sure many people who keep up with ranked charts should be familiar with. From the eclectic unapologetic patterning of *[Dream to Nightmare](https://osu.ppy.sh/beatmapsets/527728#mania/1125576)* to the creative and diverse hybrid charting of *[Luin of Celtchar](https://osu.ppy.sh/beatmapsets/1281422#mania/2879332)*, Stan has proven to be an essential member of the community over the years and has even contributed to Ranked standard sets! To any unfamiliar, this song is a chaotic and intense piano-based track based on Camellia's *Xroniel Xero* album and features a collaboration between him and the equally legendary Tsukasa Kaminose.

Onto the set, which as a chart for a collaboration between these two artists should be, is a full collaboration between a myriad of different charters. Those charters include Stan, Kyousuke-, biemote, \[Emiria\] (otherwise known as victorica_db), and myself. Besides all of us simply having GDs on the set, the catch is that each of the 5 difficulties is of a different key count. These diffs go from 4K up to 8K, and all of what's here will give players a different experience from one map to the next. From the full rice approach of the 8K to the hybrid centered patterning of the 7K and 4K, each approach to the song brings something distinct to the table. Seeing a well-constructed key-spread is rare enough these days, let alone one put together in a way that's been done here. We're thankful for Stan's contributions to the community over the years and look forward to what they have to offer in the future!
```